src/Symfony/Component/Form/ButtonBuilder.php

```diff
@@ -738,6 +748,16 @@ public function getOption(string $name, $default = null)
         return \array_key_exists($name, $this->options) ? $this->options[$name] : $default;
     }
 
+    /**
+     * Unsupported method.
+     *
+     * @throws BadMethodCallException
+     */
+    public function getIsEmptyCallback(): ?callable
+    {
+        throw new BadMethodCallException('Buttons do not support "is empty" callback.');
```

Comment:
what about returning null instead?

Reply:
Let's be consistent with all the other unsupported methods of the `ButtonBuilder`. Let's throw.